network-store-client/src/main/java/com/powsybl/network/store/client/PhaseTapChangerAdderImpl.java

```diff
@@ -158,6 +204,33 @@ public PhaseTapChanger add() {
                 .targetDeadband(targetDeadband)
                 .build();
         tapChangerParentAttributes.setPhaseTapChangerAttributes(phaseTapChangerAttributes);
+
+        Set<TapChangerAttributes> tapChangers = new HashSet<>();
```

Comment:
This piece of code is too complex. What is the purpose of the set?

Reply:
It is the same piece of code than in powsybl-core

Reply:
Yes I can see that, but we can do better here without a Set.

Reply:
Done